File: sprint4/By languages/C#/lesson1code2.cs
using System;
using System.Collections.Generic;

class Program {
    static void Main(string[] args) {
        var pagesVisitStats = new Dictionary<string, Dictionary<string, int>> {
            {"https://timofey.me", new Dictionary<string, int> {{"main", 2}, {"contacts", 3}}},
            {"https://fruit-shop.ru", new Dictionary<string, int> {{"apples", 12}, {"plums", 10}, {"pears", 20}, {"contacts", 15}}}
        };

        var siteStats = pagesVisitStats["https://timofey.me"];
    }
}

File: sprint4/By languages/Java/lesson1code2.java
// Словарь "статистика посещения страниц"
Map<String, Map<String, Integer>> pagesVisitStats = Map.of(
    "https://timofey.me", Map.of(
        "main", 2,
        "contacts", 3
    ),
    "https://fruit-shop.ru", Map.of(
        "apples", 12,
        "plums", 10,
        "pears", 20,
        "contacts", 15
    )
);

Map<String, Integer> siteStats = pagesVisitStats.get("https://timofey.me");
